python_modules/libraries/dagster-dg/dagster_dg/check.py

```diff
@@ -71,7 +71,7 @@ def check_yaml(
         ):
             continue
 
-        component_path = component_dir / "component.yaml"
```

Comment:
seems like this should share code with the core runtime and live in `dagster_shared`

Reply:
I prefer to address this kind of refactor in a followup

Reply:
100%. just flagging it here